src/main/sensors/barometer.c

```diff
@@ -142,10 +141,10 @@ void pgResetFn_barometerConfig(barometerConfig_t *barometerConfig)
 static uint16_t calibratingB = 0;      // baro calibration = get new ground pressure value
 static int32_t baroPressure = 0;
 static int32_t baroTemperature = 0;
+static float baroGroundAltitude = 0;
```

Comment:
It would be great to use meters with float value (but deeper refactoring is needed)